### PR TITLE
Fix the limit-height.js script

### PR DESCRIPTION
**Pull request by leofeyer**
Always resize all elements but only add a toggle to those elements where the content height exceeds the element height.

### CODE REVIEW

File: core-bundle/public/limit-height.js [hunk 1]
```diff
@@ -18,21 +18,18 @@ window.addEventListener('DOMContentLoaded', function () {
         // Return if there is no height value
         if (!hgt) return;
 
-        const toggler = document.createElement('div');
-        toggler.classList.add('limit_toggler');
+        const height = div.offsetHeight;
+
+        // Resize the element
+        div.style.height = hgt+'px';
```
Comment (aschempp) [inline core-bundle/public/limit-height.js:24]: should this be skipped as well if the element is high enough?

Comment (leofeyer) [reply core-bundle/public/limit-height.js:24]: No, if an element has `.h40`, it shall be 40px high. And only if its content is higher, a toggle shall be added additionally.

Comment (leofeyer) [reply core-bundle/public/limit-height.js:24]: ### Before



### After

Comment (aschempp) [reply core-bundle/public/limit-height.js:24]: Strange, but I see that was already the case currently.

**Review by aschempp — APPROVED** [review]

**Pull request merged**